js/functions.js

```diff
@@ -0,0 +1,52 @@
+const checkStringLength = function (checkString, i) {
+  if (checkString.length <= i) {
+    return true;
+  } else {
+    return false;
+  }
+};
+checkStringLength('проверяемая строка', 20);
+checkStringLength('проверяемая строка', 18);
+checkStringLength('проверяемая строка', 10);
+
+function checkWord(word) {
+  const splitString = word.split('');
+  const reverseArray = splitString.reverse();
+  const joinArray = reverseArray.join('');
```

Comment:
Методы массива в данном случае в обычной практике пишут через точку или в строку или если длинная записть то в столбик
```suggestion
  const reversedString = reverseArray.split('').reverse().join('');
```